controllers/nutanixcluster_controller.go

```diff
@@ -250,8 +263,8 @@ func (r *NutanixClusterReconciler) reconcileNormal(rctx *nctx.ClusterContext) (r
 
 	err := r.reconcileCategories(rctx)
 	if err != nil {
-		errorMsg := fmt.Errorf("failed to reconcile categories for cluster %s: %v", rctx.Cluster.Name, err)
-		rctx.SetFailureStatus(capierrors.CreateClusterError, errorMsg)
+		log.Error(err, "error occurred while reconciling categories")
```

Comment:
Nit: Better if we save the info like it was: ... for cluster %s: %v", rctx.Cluster.Name, err